tests/test_serializable.py

```diff
@@ -204,10 +208,9 @@ def test_serializable_sedes_inference(type_1_a, type_1_b, type_2):
 def test_serializable_invalid_serialization_value(type_1_a, type_1_b, type_2):
     with pytest.raises(SerializationError):
         RLPType1.serialize(type_2)
-    with pytest.raises(SerializationError):
         RLPType2.serialize(type_1_a)
```

Comment:
Removing the `with pytest.raises`... line looks wring to me, I don't think it tests the later calls. I'll submit an fix.